profile_test.go

```diff
@@ -0,0 +1,281 @@
+package eat
```

Comment:
I suggest adding tests for the examples in https://tools.ietf.org/html/draft-ietf-cbor-tags-oid-05#section-3

Reply:
In fact, I think the current implementation is not conformant: you need to strip out the TL part from the BER encoding and just leave the V, which gets packed into a CBOR bstr

Reply:
If we strip out the TL how would the ASN1 decoder at the other end would work, if it is to be a valid ASN1 payload?? ?

Reply:
it needs to synthesise the TL from the CBOR heading

Reply:
I made a small test:
```
func TestProfile_CBORMarshal_OID_draft_cbor_tags_oid_example_3_1(t *testing.T) {
	// Encoding of the SHA-256 OID, from Section 3.1 of
	// draft-ietf-cbor-tags-oid-05
	p, _ := NewProfile("2.16.840.1.101.3.4.2.1")
	expected := []byte{0xd8, 0x6f, 0x49, 0x60, 0x86, 0x48, 0x01, 0x65, 0x03, 0x04, 0x02, 0x01}
	actual, err := p.MarshalCBOR()
	assert.Nil(t, err)
	assert.Equal(t, expected, actual)
}
```
and it fails

Reply:
have you stripped out TL from the expected?

Reply:
yep, I've removed `06 09` from the BER and substituted with CBOR `D8 6F 49`

Reply:
Please note the above example has Incorrect expected byte values, this has now been corrected and a sample 3.1 test has been added!